src/test/regress/expected/alter_database_from_nonmain_db.out

```diff
@@ -0,0 +1,664 @@
+SET citus.superuser TO 'postgres';
```

Comment:
Throughout this file, it would be nice to add some comments about the test objectives.  Some of the tests seem almost the same ( I noted a below). It would be nice to have comments if it is on-purpose.

Reply:
I removed some tests and added some comments. I hope that it is leaner